impeller/entity/contents/solid_color_contents.cc

```diff
@@ -60,7 +63,8 @@ bool SolidColorContents::Render(const ContentContext& renderer,
       };
   return ColorSourceContents::DrawGeometry<VS>(
       renderer, entity, pass, pipeline_callback, frame_info,
-      [](RenderPass& pass) {
+      [&frag_info, &host_buffer](RenderPass& pass) {
```

Comment:
Note: This callback is used synchronously with DrawGeometry, so the fact it is capturing references looks good.